operator/pkg/translate/translate.go

```diff
@@ -536,6 +536,8 @@ func (t *Translator) TranslateHelmValues(iop *v1alpha1.IstioOperatorSpec, compon
 			return "", fmt.Errorf("component value isn't a map")
 		}
 		finalVals := map[string]any{}
+		// These get access to globals
+		finalVals["global"] = mergedVals["global"]
```

Comment:
I would still prefer `istioctl` remain entirely ignorant of Helm value topology, and in general it simply shouldn't do anything _at all_ with Helm values except pass them thru them to the chart - there shouldn't _be_ behavioral differences, except for back-compat concerns.

This is still better than what was removed here tho: https://github.com/istio/istio/pull/51749/files#diff-01bad0e5aa853270458b1ce81e2f93f5da901115fcf6bcb3281d9b42df43601c

and crucially it doesn't impose restrictions on Helm that make doing https://github.com/istio/istio/issues/52034 harder, so I'm fine with it.